crates/lang/macro/src/lib.rs

```diff
@@ -483,7 +483,7 @@ pub fn selector_bytes(input: TokenStream) -> TokenStream {
 ///         #[ink(message, payable)]
 ///         pub fn fund(&self) {
 ///             let caller = self.env().caller();
-///             let value = self.env().transferred_balance();
+///             let value = self.env().transferred_value();
```

Comment:
isn't `transferred_balance` the better or more concrete name actually? why was `transferred_value` chosen over `transferred_balance` if what we handle with are `Balance` types?